docs/en/user_guides/yolov5_tutorial.md

```diff
@@ -1 +1,211 @@
+```
+
+NOTICE: Theoretically setting the `strict_load` initialization parameter of `EMAHook` to `False` makes the command to load pre-trained weight matches without errors `custom_hooks.0.strict_load=False`. Since MMEngine is iterating rapidly, this is currently a problem. So for now, the only way to load the pretrained weights correctly is to turn off the use of `custom_hooks` with the command `custom_hooks=None`. This issue is expected to be fixed in the next release.
+
+3. Freeze Backbone and start training
+
+Freeze the 4 stages of backbone by setting `model.backbone.frozen_stages=4` via command line or explicit modification of config file.
+
+```shell
+# Set model.backbone.frozen_stages=4 on the command line
+cd mmyolo
+python tools/train.py configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py \
+                      --cfg-options load_from='yolov5_s-v61_syncbn_fast_8xb16-300e_coco_20220918_084700-86e02187.pth' model.backbone.frozen_stages=4
+```
+
+### Related to visualization
+
+#### Visualization of validation
+
+We modify the `visualization` of `default hook` in `configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py`, and set `draw` to `True` and `interval` to `2`.
+
+```shell
+default_hooks = dict(
+    logger=dict(interval=1),
+    visualization=dict(draw=True, interval=2),
+)
+```
+
+Re-run the following training command. During the validation evaluation, every `interval` image will save a puzzle of the annotation and prediction results to the `work_dirs/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon/{timestamp}/vis_data/vis_image` file Clamped.
+
+```shell
+python tools/train.py configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py
+```
+
+<div align=center>
+<img src="https://moonstarimg.oss-cn-hangzhou.aliyuncs.com/img/20220920094007.png" alt="image"/>
+</div>
+
+#### Visualization with wandb backend
+
+MMEngine supports various kind of backends, such as local, TensorBoard, and wandb. This section uses wandb as an example to show the visualization of loss and other data.
+
+It is required to register on wandb official website and get wandb API Keys at https://wandb.ai/settings.
+
+<div align=center>
+<img src="https://cdn.vansin.top/img/20220913212628.png" alt="image"/>
+</div>
+
+```shell
+pip install wandb
+# After running wandb login, enter the API Keys obtained above, and the login is successful.
+wandb login
+```
+
+Add wandb configuration in `configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py`.
+
+```python
+visualizer = dict(vis_backends = [dict(type='LocalVisBackend'), dict(type='WandbVisBackend')])
+```
+
+Re-run the training command to see data visualizations such as loss, learning rate, and coco/bbox_mAP in the web link prompted on the command line.
+
+```shell
+python tools/train.py configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py
+```
+
+<div align=center>
+<img src="https://cdn.vansin.top/img/20220913213221.png" alt="image"/>
+</div>
+
```

Comment:
```suggestion

##### Visualization with tensorboard backend

Install tensorboard

\```shell
pip install tensorboard
\```

As above, add `tensorboard` configuration in configuration file `configs/yolov5/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon.py`:

\```python
visualizer = dict(vis_backends=[dict(type='LocalVisBackend'),dict(type='TensorboardVisBackend')])
\```

Re-run the training command and tensorboard files will be generated in the visualization folder  `work_dirs/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon/{timestamp}/vis_data`.

Run the following command to see data visualizations of tensorboard such as loss, learning rate and coco/bbox using web browser:

\```shell
tensorboard --logdir=work_dirs/yolov5_s-v61_syncbn_fast_1xb4-300e_balloon
\```

```

Reply:
After you accept this suggestion, pls do remember change " \\\`\`\` " -> " ``` "